### DISCUSSION TITLE
Layout & Responsive

### DISCUSSION THREAD

**Discussion by zokernguyen** [reduxjs/redux-devtools#1359] [General]
I'm new with Redux devtools and having problems with its UI. It does not support responsive, so to view the Redux tab fully, I have to en-large the Chrome devtools panel, which means the remain area to view my webapp is tiny. As far as I know, old version of Redux devtools has a vertical layout, which could help resolve the problem, but with the current version (3.0.7), the layout is horizontal by default and there is no appearance setting to change it. Hope you guys will bring back the vertical layout in future updates.

**Comment by Methuselah96** [comment]
This is being tracked at https://github.com/reduxjs/redux-devtools/issues/1294.

**Reply by zokernguyen** [reply]
Thanks for responding, I'm glad to hear that. Meanwhile, can I have a package of old version that has vertical layout to use when waiting for the UI updates?

**Reply by Methuselah96** [reply]
Each [release](https://github.com/reduxjs/redux-devtools/releases) of the Redux DevTools Extension should have the extensions in a zip for each browser under Assets. I'm not sure in which version it broke.
